tqp/topic_queue_poller.py

```diff
@@ -17,17 +17,23 @@ def _jsonify_dictionary(dictionary):
 # -----------------------------------------------------------------------------
 
 
-def create_queue(queue_name, **kwargs):
+def create_queue(queue_name, tags=None, **kwargs):
+    tags = tags or {}
     sqs = boto3.resource("sqs")
 
-    def _create_queue(name, attributes):
+    def _create_queue(name, attributes, extra_tags=None):
+        extra_tags = extra_tags or {}
+
         return sqs.create_queue(
-            QueueName=name, Attributes=_jsonify_dictionary(attributes),
+            QueueName=name,
+            Attributes=_jsonify_dictionary(attributes),
+            tags={"TQP": "true", **extra_tags, **tags},
```

Comment:
right casing, unfortunately

Reply:
dare i ask why?

Reply:
I was referring to `tags` because it's a different convention than the other props. you are probably referring to `TQP`. I can make both TQP and DLQ lower case, it's probably better